cozy.example.yaml

```diff
@@ -345,21 +345,21 @@ registries:
 
 # Wizard used for moving a Cozy from one place/hoster to another
 move:
-  url: https://move.cozycloud.cc/
+  url: https://move.example.org/
```

Comment:
Well, in that case, `move.cozycloud.cc` is fine and should be kept

Reply:
I was going to but the flagship config has a `move_url` attribute with a different URL (see https://github.com/cozy/cozy-stack/pull/4500/files#diff-accceeea297d6b4bddb571eaab50129c26e0485c18c259d7822b192f3bc3df65L494) so I figured the move wizard did not have to be https://move.cozycloud.cc.

Reply:
It can be overloaded by context, but it is not used

Reply:
OK.